orca-queue/src/main/kotlin/com/netflix/spinnaker/orca/q/handler/CompleteExecutionHandler.kt

```diff
@@ -57,6 +60,10 @@ class CompleteExecutionHandler(
           publisher.publishEvent(
             ExecutionComplete(this, message.executionType, message.executionId, status)
           )
+          registry.counter(completedId.withTags(
+            "status", status.name,
+            "executionType", execution.type.name
```

Comment:
Thoughts on adding `application` to this metric? (we have it on `executions.started`)

Reply:
I had a reason for not including it, but I can't remember, so it was probably rubbish.